src/app/carbon-estimation/carbon-estimation.component.ts

```diff
@@ -1,125 +1,79 @@
-import { ChangeDetectorRef, Component, ElementRef, OnDestroy, OnInit, ViewChild, computed, input } from '@angular/core';
-import { CarbonEstimation } from '../types/carbon-estimator';
-import { NumberObject, sumValues } from '../utils/number-object';
-import { ApexAxisChartSeries, ChartComponent, NgApexchartsModule } from 'ng-apexcharts';
-
-import { startCase } from 'lodash-es';
-import {
-  EmissionsColours,
-  EmissionsLabels,
-  SVG,
-  getBaseChartOptions,
-  estimatorHeights,
-  tooltipFormatter,
-  placeholderData,
-  ApexChartSeriesItem,
-  ApexChartDataItem,
-} from './carbon-estimation.constants';
+import { ChangeDetectorRef, Component, computed, ElementRef, input, OnDestroy, OnInit, ViewChild } from '@angular/core';
```

Comment:
```suggestion
import { ChangeDetectorRef, Component, ElementRef, input, OnDestroy, OnInit, ViewChild } from '@angular/core';
```
I think `computed` isn't being used now.